lib/multi-acc-new-eks-mixed-observability-pattern/GrafanaOperatorConstruct.ts

```diff
@@ -0,0 +1,53 @@
+import { Construct } from 'constructs';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import { ObservabilityBuilder } from '@aws-quickstart/eks-blueprints';
+import { GrafanaOperatorSecretAddon } from './grafanaoperatorsecretaddon';
+
+export default class GrafanaOperatorConstruct {
```

Comment:
All our filenames are all sentence cases with dash and not mixed cases.

Reply:
done